recipes/discord/webview.js

```diff
@@ -52,6 +52,11 @@ module.exports = (Ferdium, settings) => {
           event.preventDefault();
           event.stopPropagation();
 
+          if (url.includes('discordapp.com/attachments/')) {
+            // Always open file downloads in Ferdium, rather than the external browser
+            window.location.href = url;
+            return;
+          }
           if (settings.trapLinkClicks === true) {
```

Comment:
So we don't duplicate code can we make this line as follows ?
`if (url.includes('discordapp.com/attachments/') || settings.trapLinkClicks === true) {`